File: Unicatólica/Semestre_2024_1/POO - Programação Orientada a Objetos/Turma B/POO2024B/Aula09_03042024/src/revisao2/Exemplo4.java
package revisao2;

import java.util.Scanner;

public class Exemplo4 {

	public static void main(String[] args) {
		int vetor[] = {20,15,70,9,8};
		int pesquisa = 0;
		boolean encontrado = false;
		Scanner sc = new Scanner(System.in);
		System.out.println("Digite um valor: ");
		pesquisa = sc.nextInt();
		for(int i = 0; i < vetor.length; i++) {
			if(pesquisa == vetor[i]) {
				encontrado = true;
				break;
			}
		}
		if(encontrado == true) {
			System.out.println("Valor encontrado");
		}else{
			System.out.println("Valor não encontrado");
		}
	}
}

File: Unicatólica/Semestre_2024_2/Estrutura_de_Dados/Turma B/Codigos/aula_25092024/exemplo4.cpp
#include <iostream>
using namespace std;

int main(){
    int vetor[5];
    int soma = 0;

    for(int i = 0; i < 5; i++){
        cout << "Digite um valor: ";
        cin >> vetor[i];
    }

    for(int i = 0; i < 5; i++){
        soma = soma + vetor[i];
    }
    cout << "Somatorio: " << soma << endl;
    return 0;
}